plugin/src/main/java/org/opensearch/ml/model/MLModelManager.java

```diff
@@ -818,7 +815,7 @@ private void deployModelAfterRegistering(MLRegisterModelInput registerModelInput
         client.execute(MLDeployModelAction.INSTANCE, request, listener);
     }
 
-    private void deleteModel(String modelId) {
+    private void deleteModel(String modelId, String modelGroupID, Boolean isThisVersionCreatingModelGroup) {
```

Comment:
I would suggest to add comment in the code to explain this use case.

Reply:
yeah sure